crates/biome_js_formatter/tests/specs/prettier/js/multiparser-html/lit-html.js.snap

```diff
@@ -192,15 +191,11 @@ ${ foo}:${bar};
 -    const tpl = html\`<div>\${innerExpr(1)} ${outerExpr(2)}</div>\`;
 -  </script>`;
 +const trickyParens = html`<script> f((${expr}) / 2); </script>`;
-+const nestedFun = /* HTML */ `${outerExpr(
-+  1,
-+)} <script>const tpl = html\`<div>\${innerExpr( 1 )} ${outerExpr(
-+  2,
-+)}</div>\`</script>`;
++const nestedFun = /* HTML */ `${outerExpr(1)} <script>const tpl = html\`<div>\${innerExpr( 1 )} ${outerExpr(2)}</div>\`</script>`;
```

Comment:
All mismatches occur in this file because `Prettier` treats these template strings as `HTML` since the template string has the leading comment `/* HTML */` but `Biome` treats them as normal template strings.